docs/docs/nessie.md

```diff
@@ -93,7 +93,7 @@ table_env = StreamTableEnvironment.create(env)
 
 table_env.execute_sql("CREATE CATALOG nessie_catalog WITH ("
                       "'type'='iceberg', "
-                      "'type'='nessie', "
+                      "'catalog-type'='nessie', "
```

Comment:
L102 still mentions `type`. Can we update the line too? 

> settings for the `type` or `catalog-impl` and the required config to start Nessie correctly.

Reply:
> L102 still mentions `type`. Can we update the line too?
> 
> > settings for the `type` or `catalog-impl` and the required config to start Nessie correctly.

I move up this line, it's for spark.